packages/contracts/utils/ens.ts

```diff
@@ -34,6 +34,14 @@ export async function setupENS(domains: string[], hre: EHRE): Promise<any> {
   const resolver = await deployments.get('PublicResolver');
 
   for (let i = 0; i < domains.length; i++) {
+    console.log(`Registering subdomain ${domains[i]}`);
+
+    const resolved = await ens.resolver(ensDomainHash(domains[i]));
```

Comment:
Should we add ENS related contracts to `verify` also?
so their contracts will verified on etherscan ?

Reply:
they are added in `packages/contracts/deploy/10_framework/01_ens_conclude.ts`